plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -246,6 +350,22 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 	}
 	</#if>
 
+	<#if data.isWanderingTraderType()>
+	@Override protected SoundEvent getTradeUpdatedSound(boolean hasContent) {

```

Comment:
Following up on tests we did with @Defeatomizer, those checks are no longer needed as intentionally_empty is the default mapping. Applies to both versions.

Reference: https://github.com/MCreator/MCreator/commit/994e09d4dac393bdb77efc398fb0540fdd746df9

Reply:
Done!